apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/OptionalAssertionsSpec.kt

```diff
@@ -33,5 +33,9 @@ class OptionalAssertionsSpec : OptionalAssertionsSpec(
         o1 = o1 toBe Empty
         o1b = o1b toBe Empty
         star = star toBe Empty
+        o1 toBe Present
+        o1b toBe Present
```

Comment:
```suggestion
        o1b toBe Present
        star toBe Present
```

Reply:
Hm, `star toBe Present` doesn't compile because required type i Empty but passed one is Present. Should I add `.toBe` function for `Optional<*>` like it's done for `toBe(Empty)`. Seems the same problem is actual for `fluent` module

Reply:
ah right, no, I would say we don't support this. please add the following line instead
```
// following is not supported on purpose as we don't know what type the element is in this case
//star toBe Present
```
Could you please also add the same to the fluent Spec, should have been there as well:
```
o1.isPresent()
o1b.isPresent()
// following is not supported on purpose as we don't know what type the element is in this
//star.isPresent()

o1.isPresent { }
o1b.isPresent {}
// following is not supported on purpose as we don't know what type the element is in this
//star.isPresent {}
```